builder/builder-next/builder.go

```diff
@@ -97,7 +97,7 @@ type Opt struct {
 	ContainerdAddress   string
 	ContainerdNamespace string
 	Callbacks           exporter.BuildkitCallbacks
-	CDISpecDirs         []string
+	DisableCDI          bool
```

Comment:
Not for this PR (although, not a huge fan of booleans :joy:) wondering if we should have some `features` array/map at some point to allow passing-through options to BuildKit. We could even filter-those from the daemon's `features` (e.g. some common (`com.moby.buildkit`) prefix).